impeller/entity/contents/filters/gaussian_blur_filter_contents.cc

```diff
@@ -356,6 +351,9 @@ fml::StatusOr<RenderTarget> MakeDownsampleSubpass(
         SetTileMode(&linear_sampler_descriptor, renderer, tile_mode);
         linear_sampler_descriptor.mag_filter = MinMagFilter::kLinear;
         linear_sampler_descriptor.min_filter = MinMagFilter::kLinear;
+        if (min_lod_clamp) {
```

Comment:
With a downsample factor of 0.5, by default we will try to sample from mip level 1 instead of 0. But forcing 0 and using linear filtering is sufficient to not lose quality.